libs/platform/src/lib/components/form/form-group/form-field/form-field.component.ts

```diff
@@ -370,4 +393,64 @@ export class FormFieldComponent implements FormField, AfterContentInit, AfterVie
             throw new Error('fdp-form-field must contain [id] binding.');
         }
     }
+
+    /** @hidden */
+    private _setColumnNumber(): void {
+        try {
+            this._sColumnNumber = this.columnLayout['S'] || 1;
+            this._mdColumnNumber = this.columnLayout['M'] || this._sColumnNumber;
+            this._lgColumnNumber = this.columnLayout['L'] || this._mdColumnNumber;
+            this._xlColumnNumber = this.columnLayout['XL'] || this._lgColumnNumber;
+        } catch (error) {
+            this._isColumnLayoutEnabled = false;
+        }
+    }
+
+    /** @hidden listen for window resize and set value of column and isInLine based on screen size */
+    private _setLayout(): void {
+        this._setColumnNumber();
+        this._updateLayout();
+
+        if (this._isColumnLayoutEnabled) {
+            fromEvent(window, 'resize')
+                .pipe(debounceTime(50), takeUntil(this._destroyed))
+                .subscribe(() => this._updateLayout());
```

Comment:
we have to call such kind of code outside of `ngZone` and call `markForCheck` if something really has been changed

Reply:
running outside angular zone now.